src/index.jsx

```diff
@@ -25,6 +26,15 @@ const App = () => (
   </Provider>
 );
 
+const Unauthorized = () => "You are not authorized to access the gradebook for this course.";
```

Comment:
I recommend we add an action link to this element. Say `Click here` to go back or something.

Reply:
Go back where?